packages/dashboard/src/components/dashboard/tests/negotiations-panel.test.tsx

```diff
@@ -1,16 +1,15 @@
-import { createMount, createShallow } from '@material-ui/core/test-utils';
+import { createShallow } from '@material-ui/core/test-utils';
```

Comment:
Not sure if this is the correct import

Reply:
yea this is used in the very first test so I have not been successful in replicating this test using RTL. and I'm not sure if it's actually necessary?